scopes/dependencies/modules/packages-excluder/packages-excluder.docs.md

```diff
@@ -0,0 +1,18 @@
+---
+labels: ['typescript', 'utils', 'packages', 'node modules', 'node', 'regex', 'excluder']
+description: 'Create node modules regex with packages.'
+---
+
+import { generateNodeModulesPattern } from './generate-node-modules-pattern';
+
+A function that receive an array of packages name to catch in node modules and return a regex of it.  
+The returned regex can be used in Jest `transformIgnorePatterns` to ignore specific packages.
+
+For example:
+
+```js live
+() => {
+  const packagesToTransform = ['react', '@myorg', 'testing-library__dom'];
+  return generateNodeModulesPattern({ packages: packagesToTransform });
+};
+```
```

Comment:
add another example which uses the regex (like in the tests) and return true and false for different pacakges

Reply:
done